docs/developer-docs/multi-chain/ethereum/evm-rpc/evm-rpc-canister.mdx

```diff
@@ -835,6 +837,33 @@ dfx canister call evm_rpc eth_sendRawTransaction "(variant {$RPC_SOURCE}, $RPC_C
 </TabItem>
 </AdornedTabs>
 
+
+### Error messages
+
+#### Error "already known"
+
+```
+"{ code: -32603, message: \"already known\" }"
+```
+
+If you receive this error, but your transaction can be seen on Ethereum, then this error is expected because HTTPS outcalls are replicated meaning only one of the contacted nodes from the JSON-RPC provide should return the success results, while others return an error indicating the transaction is already known.
+
+#### Error "TooFewCycles"
+
+```
+"ProviderError(TooFewCycles { expected: 798336000, received: 307392000 })"
+```
+
+You may receive this error if multiple JSON-RPC providers were queried and one did not receive enough cycles for the request. You can safely attach more cycles to your query, since unused cycles will be refunded.
+
+#### Error "Your app has exceeded its compute units per second capacity"
```

Comment:
I'm not sure I would put this error yet since we are internally discussing whether we could upgrade the current Alchemy tier we are at. WDYT @jasonaibrahim ?